src/pip/_internal/resolution/resolvelib/found_candidates.py

```diff
@@ -9,20 +9,62 @@
 """
 
 import functools
-import itertools
 
 from pip._vendor.six.moves import collections_abc  # type: ignore
 
 from pip._internal.utils.typing import MYPY_CHECK_RUNNING
 
 if MYPY_CHECK_RUNNING:
-    from typing import Callable, Iterator, Optional
+    from typing import Callable, Iterator, Optional, Set, Tuple
+
+    from pip._vendor.packaging.version import _BaseVersion
 
     from .base import Candidate
 
+    IndexCandidateInfo = Tuple[_BaseVersion, Callable[[], Optional[Candidate]]]
+
+
+def _iter_built(infos):
+    # type: (Iterator[IndexCandidateInfo]) -> Iterator[Candidate]
+    """Iterator for ``FoundCandidates``.
+
+    This iterator is used the package is not already installed. Candidates
+    from index come later in their normal ordering.
```

Comment:
Minor typo. I believe **when** is missing –" _... is used **when** the .._."

Reply:
A pull request would be welcomed :)